### PR TITLE
SEO-177700-JS-dropped-keywords-issue-fixes

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @gowthamanthavasiyappan ,
Please review the changes.
Thanks.
Regards,
Edith.

### CODE REVIEW

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/DateTimePicker/Getting-Started.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with JavaScript DateTime Picker | Syncfusion
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply js/DateTimePicker/Getting-Started.md]: resolved

File: js/DateTimePicker/Getting-Started.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+description: Learn here about getting started with Syncfusion Essential JavaScript DateTime PIcker Control, its elements, and more details.
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply js/DateTimePicker/Getting-Started.md]: resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Pull request merged**